guides/source/active_record_validations.md

```diff
@@ -1055,7 +1055,9 @@ person.errors[:name]
 
 ### `errors.add`
 
-The `add` method lets you manually add messages that are related to particular attributes. You can use the `errors.full_messages` or `errors.to_a` methods to view the messages in the form they might be displayed to a user. Those particular messages get the attribute name prepended (and capitalized). `add` receives the name of the attribute you want to add the message to, and the message itself.
+The `add` method lets you manually add messages that are related to particular attributes. It takes as arguments the name of the attribute receiving the message, and the message itself.
```

Comment:
I find the explanation for the first argument tough to understand as it's very abstract. Is there a way to make it more concrete?